src/pages/projectSettings/Header.tsx

```diff
@@ -133,22 +133,23 @@ export const Header: React.VFC<Props> = ({
             Save Changes on Page
           </Button>
         )}
-        {projectType === ProjectType.AttachedProject && (
-          <>
-            <Button
-              onClick={() => setDefaultModalOpen(true)}
-              data-cy="default-to-repo-button"
-            >
-              Default to Repo on Page
-            </Button>
-            <DefaultSectionToRepoModal
-              handleClose={() => setDefaultModalOpen(false)}
-              open={defaultModalOpen}
-              projectId={id}
-              section={mapRouteToSection[tab]}
-            />
-          </>
-        )}
+        {projectType === ProjectType.AttachedProject &&
+          tab !== ProjectSettingsTabRoutes.EventLog && (
```

Comment:
We can use the `saveable` prop passed into this component instead of special-casing for the event log tab! It's based on [this list of read-only tabs](https://github.com/evergreen-ci/spruce/blob/dea67742353e91c21dffc2dd3b82b40f178f3d1f/src/pages/projectSettings/tabs/types.ts#L30) (which right now is just the event log).